ui/src/audioplayer/Player.js

```diff
@@ -48,13 +51,17 @@ const useStyle = makeStyles(
     },
     player: {
       display: (props) => (props.visible ? 'block' : 'none'),
-    },
-    artistAlbum: {
-      marginTop: '2px',
+      '& .progress-bar-content': {
+        display: 'flex',
+        flexDirection: 'column',
+      },
       '& .play-mode-title': {
```

Comment:
This block of code should be inside `player` to work.

Reply:
Yes, this was the result of a bad merge. Thanks for fixing it.